bokchoi/scripts/gcp-startup-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ "$(whoami)" != "root" ]
+then
+    sudo su
+fi
+
+apt-get update \
+    && apt-get -y upgrade \
+    && apt-get install -y python3-pip python3-dev unzip \
+    && pip3 install --upgrade pip
+
+BUCKET_NAME=$(curl http://metadata/computeMetadata/v1/instance/attributes/bucket_name -H "Metadata-Flavor: Google")
+PACKAGE_NAME=$(curl http://metadata/computeMetadata/v1/instance/attributes/package_name -H "Metadata-Flavor: Google")
+ENTRYPOINT=$(curl http://metadata/computeMetadata/v1/instance/attributes/entry_point -H "Metadata-Flavor: Google")
+
+gsutil cp gs://${BUCKET_NAME}/${PACKAGE_NAME} .
+unzip ${PACKAGE_NAME}
+
+pip3.4 install -r requirements.txt
+python3.4 ${ENTRYPOINT}
```

Comment:
Does the instance shut down automatically after this? Or do you have to kill it yourself?

Reply:
it does now :)